test/extended/util/annotate/generated/zz_generated.annotations.go

```diff
@@ -1401,6 +1401,8 @@ var Annotations = map[string]string{
 
 	"[sig-network][endpoints] admission [apigroup:config.openshift.io] blocks manual creation of Endpoints pointing to the cluster or service network": " [Suite:openshift/conformance/parallel]",
 
+	"[sig-node-tuning] NTO should OCP-66086 NTO Prevent from stalld continually restarting [slow]": " [Suite:openshift/conformance/parallel]",
```

Comment:
the `[Slow]` keyword is case sensitive, since you marked it `[slow]` instead, it got picked up for the standard suite.  Please set it back to `[Slow]`

Reply:
Fixed in new code, thanks!

Reply:
But It seem still not pick in standard suite.

Reply:
right, it should not be picked up in the standard suite because it is marked `[Slow]`, and that is how we want it to be currently because we don't want to add a test that takes 10 minutes to our standard suite.

I think you said earlier that the consumer of this test is going to ensure it is explicitly run in their test job, right?